components/director/internal/domain/api/service_test.go

```diff
@@ -371,7 +371,7 @@ func TestService_CreateToPackage(t *testing.T) {
 			},
 			FetchRequestServiceFn: func() *automock.FetchRequestService {
 				svc := &automock.FetchRequestService{}
-				svc.On("HandleAPISpec", ctx, fixModelFetchRequest(frID, frURL, timestamp)).Return(nil)
+				svc.On("HandleSpec", ctx, fixModelFetchRequest(frID, frURL, timestamp)).Return(nil)
```

Comment:
Since you're removing the `API` from the function definitions, remove it also from the name of test functions, ex: `TestService_RefetchAPISpec`. I also saw it in the fetch request service tests and I think also the events service tests.